db_revisions/versions/b3bfb504ae7e_remove_confirmation_id_from_submission_.py

```diff
@@ -0,0 +1,27 @@
+"""remove confirmation id to submission table
+
+Revision ID: b3bfb504ae7e
+Revises: 8eaef8ce4c23
+Create Date: 2024-03-14 03:18:39.063892
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "b3bfb504ae7e"
+down_revision: Union[str, None] = "8eaef8ce4c23"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.drop_column("submission", "confirmation_id")
+
+
+def downgrade() -> None:
+    op.add_column("submission", sa.Column("confirmation_id", sa.String))
```

Comment:
I think the add and drop can be in one script.  No need to separate them out for this.  Otherwise looks good.

Reply:
OK, changing it.